feature/home/src/main/java/com/moya/funch/navigation/HomeNavigation.kt

```diff
@@ -0,0 +1,35 @@
+package com.moya.funch.navigation
+
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import androidx.navigation.navigation
+import com.moya.funch.HomeRoute
+
+const val HOME_GRAPH_ROUTE = "home_graph"
+
+fun NavController.navigateToHome() =
+    navigate(Home.FunchHome.route) {
+        popUpTo(graph.id)
+    }
+
+fun NavGraphBuilder.homeGraph(
+    onNavigateToMyProfile: () -> Unit,
+    onNavigateToMatching: () -> Unit,
+) {
+    navigation(
+        route = HOME_GRAPH_ROUTE,
+        startDestination = Home.FunchHome.route,
+    ) {
+        composable(route = Home.FunchHome.route) {
+            HomeRoute(
+                onNavigateToMatching = onNavigateToMatching,
+                onNavigateToMyProfile = onNavigateToMyProfile,
+            )
+        }
+    }
+}
```

Comment:
```suggestion
fun NavGraphBuilder.homeScreen(
    onNavigateToMyProfile: () -> Unit,
    onNavigateToMatching: () -> Unit,
) {
        composable(route = Home.FunchHome.route) {
            HomeRoute(
                onNavigateToMatching = onNavigateToMatching,
                onNavigateToMyProfile = onNavigateToMyProfile,
            )
        
    }
}
```

현재 home에는 하나의 screen밖에 없기때문에 nested Graph로 만들 필요가 없다고 생각합니당